src/v/redpanda/admin_server.cc

```diff
@@ -904,8 +904,7 @@ void admin_server::register_config_routes() {
 
     register_route<superuser>(
       ss::httpd::config_json::set_log_level,
-      [this](std::unique_ptr<ss::httpd::request> req)
-        -> ss::future<ss::json::json_return_type> {
+      [this](std::unique_ptr<ss::httpd::request> req) {
           ss::sstring name;
           if (!ss::httpd::connection::url_decode(req->param["name"], name)) {
               throw ss::httpd::bad_param_exception(fmt::format(
```

Comment:
nice. the futurize_invoke is what i had been missing when converting these earlier to deal with these throws. now it's easy.

Reply:
It's not quite the same though, as any intermediate exception handling is bypassed (there probably isn't any here).